mage_ai/server/websocket.py

```diff
@@ -50,35 +50,46 @@ def on_message(self, raw_message):
         message = json.loads(raw_message)
         custom_code = message.get('code')
         output = message.get('output')
+        if output:
+            self.send_message(output)
+            return
         global_vars = message.get('global_vars')
         execute_pipeline = message.get('execute_pipeline')
         kernel_name = message.get('kernel_name', DEFAULT_KERNEL_NAME)
 
         run_downstream = message.get('run_downstream')
         run_upstream = message.get('run_upstream')
+        run_tests = message.get('run_tests')
+        block_type = message.get('type')
```

Comment:
You can remove the duplicate code of get type, uuid, pipeline_uuid below at line 103